### ISSUE TITLE
Automator Crash in Fast Realities 

### ISSUE THREAD

**Issue by lrobt97** [antimatter-dimensions/mobile-issues#505]
**Describe the bug**
Attempting to edit automator scripts while fast realities are running can cause the game to crash

**To Reproduce**
1. Import the given save and run the given script 
2. Attempt to change one of the lines while the script is running 
3. The game crashes 

**Expected behavior**
Script stops execution while editing

**Savefile**
```
AntimatterDimensionsAndroidSaveFormatAAAH4sIAAAAAAAAAO0a9XXfbNrJ0cpcfPqg40bSaBvjqOmOadufW1n9yHrB0aaCJa4pUZeiknjT0cPc7A5Ag0bCFLsp222eskTkIQGAwGg8HMYAb8cpJMF6n5aJZmVW5OfnrPpBw96YeOyM3oJFmV6TIpS1Oc0cPTlxHxe5yvo4OQnysnoZIlvN5vk5Cc5Jl0bh8rbMsRqgcp0ausUmsRUzHULWNXVlsDcJepYs0aS8o0aX2Gz5TYr0a3WWYkM5Okk3p9My0cWhcdQS0cmZrVpqqcJZvyOp3enfxEEH6v8od0aflZsV9OFRWiZbxHrcARaSB0aOgY45NFvmM4DBRicwbIOwAZHPv6Tzhdl0a2jPSbo0aEmOZYi7M4UhapFdDtY5LBa2Kx3In0abZJtLNZZMr37Jc0cMRf4JyfD0bS29kvRKYpVlS3J3bWXqdLjeHkGdW0af3del4kM7OxXTWN1FhRoYkQgmoZU8UmdADKn9ACB5cuX0bW5I0c0bH7f158vmtpysMDiZonmTJ5xQHEQGN0a2Vavq7aeF6zpe9gtrI3vrIlOk7i56C6qGeRyei4OfSUXToudTT9sM2gqqTMVX63yvLpnZnVeDl0bw4EXJpndX0beugkfO4SKApchjsHnp0c1n6V0aLTl0c5f0bj0b2fyu9NsmHzJzlIHtW20b3SQzbTGgczrXZLOWZCK8kZITHVVE90bVPB2fQ77oq8L0ca0cS8n7XrtbalIjdlDpbGkc8d2xp0cR2tv0aVZYXu0cWxhbSTskZJsy6mUsEUdulJWQnWf368UpqBtnmUlWNXFs6VmeZcl6Y3zXtvQqL3BnBlDLZJ6ukuK0btfPt3WT0cbiU3SLTbFFnBbfnv90b750cz8qIGFaKwbWdVYtGMsJqLvGkvilDzWW7WUQtRYRG8vo6wALVnD0cDDa6wd5AMZ4iUjs0atm9GzU0ayLdLbdGq6KvpOscM6kqMi2CZdzbdZUlST0aXtfAhabtUHZ0cOUBSb1HVNPHaW0aoG2Eh1Z1Vj39btfiJLf66tYmEvV6YvDDOTunUQBMoWcGsv3XCzW4Xe0cY3b0blhXWKfJ26XTOuS2qqy0a5lgh9b4gbdMyZErMfXcz7ZFxzRCWzedWVA1YwUvYddcLxLccATlYqw1E5EUEaMR2JdrtOOCygJ29G2Ls5FsLZSEiPejxCKHEh1CiSmPkp2GLhK8j8RNQCXcVy0bS7cac2yUXEvA3M0a8cYtaqb8xVrO5XYrv42qodyKScipgTFjVM0bqFI7vwoP8C6vqsm0cv7SgHqBmhOuQTWKRxTdEa0aq1YhepWi3SYpahYH6ZQpYWK65vQVpM7eqEgzpdPbv7aZ8g3vAPw2oP41x6DSEKTDllSlBTZvb5k7gSBRcy3VmJre3ZlqeOZULUC0cv1445K1Pd9oc1FjkIS0cvG0ah30bqvuxz0au7N1WUsiVgMX5wBm5YCsOnpFG7TNi9qJ8t0ap20bSKsj0auuEtDvAJVRYqXy9AFiLPJtVYnaDwINCrFnTfnC8tUYSDLkuAv413bK3F37EdRH8u0cR79wGjbvVIut2Rdl9koJ0cjRn0cpduHaYdUjQNG8r3ayWgfx5QFtgtJZOU0cSlSdH8MbyZmY0bmuxwXuijQfo4kEEEyM7Oj6OUW0cQDJMINIKABPlY4VmPHErRyWusECycXB7OF74S0aeiBd6KSBfNzwXgduy94Qa90ca62svv6sSZzsFsNqvr60cnZtUttOM4dObDrskD0cZJep6TT43H0amDSitkeOSgzbTWFhN0bHKlHXlRWASu5JKTW8XBg7kDoDbvJjnZWlW3SahJ64DLPBqD7FU3KdsfxSkNQTSRZ8Mo0a6G8CaDOJNBfI0bVWN76GZBW9l1PwnjbZt614qvyIv9EG7b0axa0bvw5nr0apQMyakeAmSgc9LvmPQ7HaTLDqqcbYuNszN6RJnaV72Ru0bIBjnIvAtulVT65OJwa3Z7JUK9koEcS9nYUGfLlOlntEmPT0bi0aojWACrp2888ZW9fKoEQ5CJB1oh40cga0b0aJui6SzeK89nPNWyoeCMXqtMasq38D60aBUGx60bsUY9bH1Xi6SYdeyOWLV9YyxSjHD4O4oJldhy2xDH6dZOL6UsRsqsNlny0abEc6qJXJYwF12ajM6Ny5ldrQOUM0aPU7xa3Jyol3AYYOrl0cADNi8AdXcUXQBjw6B6sEZyxdFPgcCQnEsWSzUOOLa0cRLWWgPULkGHTtBW90braBrEdKmgNYF2BroatSZsmGxzxzLofwTiCbikYA4SqKJpQTev3Pxdo0bCBXWPvCFbpucK8G8AZrbKxe7siM9gEZUTAO0cmOKHMlwXR3d4YRmSVqaLLFskBR3k5Up5veVWdScmu21MZuqzrVksZult7fpFJ7vr61DhPQOf0cyJ5WaH8Vjr4q8nA2XnrTJUVK7S1dT8Cjb2u0cXMabj2bPUQP7A1M10cQ0bU7QuUGRURb5uubVW4ADpu2ZEwh2hfCocW2dLQwa80bcp2r8AdVM1w7PNw7m8tV69ty710aAIP3lmyfruawh68qUYcvENfuxUXqI9u1lkCxvwm3xZTpzSB0bP50a1gg2LIEGvxezag3hU9AAnXOLdNMfObxYmywzbmdxTn8rYPtqS3pbNk9bP1H1vjXL82WwdTtt0bY0aBWZGAALIg3YOZvQm359rHEjhJW6clUL4GWrfaNH6Xuve24Gw0am6FtaGjSQkcJGfV0bW0bcxbJ0aOVJmXSXa6I6hjALqtX0b84F3m6KvdjZNvUjoKD2thhLFeJrWlZrZoymNlFkm9cUEmLsrdplu3FZG2Ah1ZlMjdXa1fHwjaz7dTMrvP6kGtmpsn9N0b5jnQDT5Z0b0ccS8w0a6AJVhrdN0bznYzK1Z5nfsJOv4Up9f2OVQtthNl2Y5b3bXythyiQhFYxaf0cCu1ZcqL1W0bxyogt0aH2FXMzcxuDIFracAJX6lfcwp3kXxqQt9bH8I8k25rQGWT7UGwstJCcEO7dCv0aXqXfvdt0bsnZe7W1yEHtHuSxdw0aC61AQmI5nbp9PR4LEWstYr4BI8tTHL3JvEeO6tnr0a35z7RcXBrc5U1FI27Lu370b6WK7ums0aMTxiqBwLDIi3NEuwW0cCFHDMeU8qJFlzGEkw4sEKqCvdnCGRjO40ciCJAT8ItRouyRhasTHLbQjn3XxYHuwIGBEcljEdFYaqIIU5pPqOoiwcaSKRHHEWFSxpwCpnoHErBLm0a1yDBnoGGjAFZ6HR5GOtJow0akcg0akgFRiQlTCgk1S4yfDym82gM41FakYgSMEWFiAeGT8dKw0cQoppVQTEglBofPvj5ggoOVDrMspGzR6AMsmtPzK1PancS5KvBwTAnmjtYYw6lLZ7BnY1E1BuAZJmoHhT9exsOgk59a5wO4xj1UoDKNNJU1aKp5BZoeBrl2sHagAlvGseA1wqKGKsVhYO0aSfBimO2lEmPyxMGHdAwPFMM10b0cLImbcMdFF0aCuynrPCP2oBLF3XZ1et5WNARTnDOlgLgi1Dk4MDC8o0aIoIlRM1FdnM8FkGVTpz9E0cu0boeNEP3JArB6HHMYwWyTEUKJQZziszd1SJfY7w1GcEfGEBkVe41mC9mdlrJeJAkE0auwHRwa8XrxXF1XKpGAzud5kl2iVoQMdVWa9aCKf9OGC0c0b3Z5xY90c3NKORt1rA2YTVrHzap0csCjPbHfZLGEUDGizEL9e67A50bXs9lyR6oTfxgx0aaBARG7J4xFSF8GoM0bRPgacVJA1L2ifkQSHdk0cgA8rnvseSQ8WOicKqI8RNablocg0btluzwqVFTxBWV0cURhQXwcGS1sMioi9iHwkLFIoaVsAtlOhHwaoZRdLHwqJAY5C6DaPUu0ancAhkfOQ8vkux7lGTCxqV1JNkDfL5Pkgmi0b5LsSfBUX4wdDa8lyQSpJ4cdJicGRBmloPuCidDIMrlHlmk9FntFWZuNqFKeQbUHq2Nk2yPYSGB4FwheTwAwniqwOiAq0cNoFtjqb3Qe1Rlaz54Qa11D5c0aKtZZ4WR0aFtz1cjS6hmfVkCM7VTiB4qSgIeII9ngY4FAeKp5gAWh1DVo40aSIKkXJFEAcvd2PgASLAWhwbL0ay0aoL3Vv9mvLjxd3L1vQ33pqctXRZn4i80cxGrAQUkuppA5SFExmMhI6EYKMp8BMU0aVgrIFWspIuCYiR5RJWOmxlJGjMWx5qAD2mxBikC47VlTFY8r20bydP6iORs1vaDMSN71ogdHJJzwH0bTlL12sfnvvVOgOzzKzmLqwuSAOa0bpMx7EGOBn0cf2BiRwocgTdXbJn3EHXtebIspHmhtfjW3UEu6apc0b8aDz3kVO0cO82LazX0aZ9eNQdcNgsgiBdFux0aXorViMVdo0c183QQxEHedL2kNJp6yJOEmnwp0cFN4igxb0cKi2WSPRKNAazYBcYh7TzoqPKBea0cWjwNJVlN1jUdmV2GmxcNw9XUd0cnOWb8pX20bV64wNpAmIpG4bRUOtrkPkbHt5U8dcwuT8XxrTOK211M0cPxnA0bd69x2G6ObBUN6WnnX7eEoShkDe0a0cgHJnPZRDgE7bifSLsOvjEnJVyO7M4vMe4kDKZLu6Bv5OiTKdZz9EjRQcjZRUuRmIpJKW6OYgKnDU0b0aMPnDbuIkCb840asrdx4zgfPtfFFW0axRO2sERCi8Q0cmsg3ISiEZkD1l7tLSSjLvQ2NNVfC2G6VOUHPHwkR3St0c7qu6YAc0cLP6Hrrr4U8bt0chLB0c5Xdi70cys7Vc3Z0bUx0bGBvsAHeq21ZS7W1X29UA7ImkQMA1biLF8NsCtnZmN42cD3NqW1Zg0bBvDw1PIhxnqs5O0cAjr4dbDAsvh3sJ9AEY3PcLQXNiaxsh1m35pW0aWCYaa61jUOsjzRllVt8yDwScPaA0c1s3etdW18IIY6gMm0aE54ZUlyvUg3r8P0auPTw5NP0agTC3wTuSmgafHgy0co63arRFRDqYwGsAdmiPmGwyemKwa0bZQFAag0aiqNYYvh1ROzqOigEcGU0b2YHhv0c0bzNVtzkW0cSKkgb49owmguMVUdcG0ct3ZTAKHbToGCZru8GkzM1v0aPosS60aZjAp0avi4bPRnV5PopuLipTtrxxm4no79KIeup90cV9UdsyBxPBFJvffVYVli0axqnPy0aVQTBm2S4vfVZRuqL92UCV4dUSWhJ58nq7KK38DVCESBGj0bnRZC0baTPOymS5vrbOB3cx1HKZbDzsHKa1CMacwMPczGa5x9NnsgICxnfvfQCTz2nTny0b9cqhWxY2h16NjmDiABn4y9Uagy4VtLNGw6rvVLK0crLZJi0a70bBozM11WM1hjomtgPcJ3GA9XeWZ3nh6bQwwNiFL20cSIBbpzPgQ4tMmveoy0c9TE0bS7y8s7ce2DIvr0bks1mQJ7YCZnUUOrmaptAe0aTpBtMG0anr5dlflFkZdmWgb7fWHK4v6snp1OceOicX1iqPNplvnyTfgCQ8Ovzepyu9pcFAhubipHhquztav20aq5cWYWQlj4vflZFTQaFTgj5aEokfU0bExe3dWlEdwTJVVEZS2TSWJl0cMLxUHwd9SQ8dRrGJ0aymkWSy5ihiE5LqFnYzUc4O0cS9lYb4j9keTL7YV0agH0czA0crWya0bqHmkH0ctVpjnvgPZEx0c2PxrVbMpVhtsXePoM0bBXCapWJ3i9V5k5zzn0abAkHmM9SGMOVRc0cWvs2zLP80agQFtm3A7LC6mxi0c30btqaOqeosMEifvpA5ty5uyHSlfltu0cyAhOFWzcvXv6JD0andok5QSqDbD1XLaugcvXFytIOfQoRwJwYQkWtQnCrDdkZ770a4YR7nV0cPuSs5d65HDprpdp0cNrvDr8xJPOBXfgDgIc5q7g0cL0bROBsgDmkKP0bAZiNox5dgYUNI6po0cMZlI9aZ7rjL5MV9e0aYZiahUArr18QlV0cyrwZzedQ9cYICi1VgSvUFAW4fq0ayCfx7TuGqg83Y0cKch1v1dMTf5Hgv0cibHe0cE3Od6LjzveG0bZFpuppisixzLgLYn2uFR24sIchRlQ0aMHUtgfiBWPqLI3Zhqes1EB0aoLR44fW0bA1gubHYjmwPF7iGN9nBUdKHwOoaSXZwfOzjAle3Gbul5C8sChHxALqn0as6IG4DsSCBueOul6P4kAUHzh47GJaryJ5IDftwdTHox0a48J2YhnPPiV9FB2I5PPet7XHwhPgxmLZWEm0cOiA0bkwFAgS2eWOBmMLT5qllr0a9PFF0cEB4h60alTgfjqp0bAZ0a1MfiC8YTzbM0aR9mOiBM78rCtNDZDUvsQPJuR9ivY7YgQOvIN7YNElnijkjj1lDsp3mQscyUhEHWybWFBQmsEyiMNuFjgWgE4PeqyKiI0bFeN0bcw0aJ6rSHAlwSKKMLPfVqhSX0bCtVIwxwjmXmkRM2LfeYsIrQjVpftFOdgy250ayJSFKG7i7BbXvnF8auBdUxjwH1KGYqnthzxq0a1uW36W3p0blqyxJpc0aUkwqSqWkmtEJitH6Wssmmn1PxdYFW8JFI8APhd0c1MWpQt30cO371I0a79Y5h0bNi0aD0cfVXZz6EXAg1Wfz0cbbK0c3CatfpcutPYXGQkYpTCHjsJyIVtpev2aKuyEHXMfYFVzH0aAiMXUZsugq2c4fHsAHJeASKMch32IdhmYPiCVoi3qsCOwgFo4OCBKQY3TGiWAB0awtdsBCq7oiMF0c8HnaARbI0agzBhUw0cge4BX4ERpiMYBmASAGFWSAkqB1jvgDIAhqNQIkEBRpUPtDRQAGSciSjG89cO1IUEKW2Vx9s8piyGPYDO8DWETmNoSf7w9yPjG9qX0afjnDFgvM0cegD20bSdGt8yMY5TGFxUI1UZgGJfFCNJub9SMAwwsHFarV6NB4M0bw6wldX6cx8SIqLZFWZNq0aTdSAvLDnpS3vsVlVwCa51KrNNA2ncFfjYjTWIMFKoe5CD1yRqAUwgIyKEdRy7ti0b5QN9pLtD5Ml0a9vDRAvMaRRo9E0cffXoG1v4r0bNa0bQ5HTk3ldfo1woZYZXppqg3pGfxdnyfjp7vZzYv29Xsjv8ilb40cqVTdOuSmrtrGXQQdZfbCcxu6t3uNQke8t0aYfivDet0aSp8rwZBDfHbGdk9wEr9PnC0aEM8vYuBPw3ms4W20b9ANO3XWmb80aZ87nW80ap5m0cDW0aGksCei9lwnvIP40cZcqCHN60cybp7lUDx77r8DzooBYBh41jDfsbx7zBCE3NeMLRzVw0b8BYp94LgC4IvCL4g0bILg3xnBG70b0chDfkv0bwvLwi0bIPiC4AuCLwg0b10c5y2QSQvf0cSdjZRjiEFuwfCRCt2Gy0cbInjXLx6BCW6vS94zsE66FzRsDY0a7K9p0c0aCzieCUo0aRKPfsQErEik9FNxRlwZIwRv34qPx5nyNs7ue5INzhGhGDzHuMITpedBWjCpiMaPrkZRLOjxSPMO0ahHpII0a3LOE0cmAL7XDjHWmhNmMSv7D2C0aEy1cBYkbuMsleJcx8DrQtJnQ1pLWD0bKEErEY5Dmsk3omLWRJkATyiSLI0al0a9FwsTUUkOKexFIKqR6xD2lmHosPThAM0ca0a1BrPDnojRVjChNI65ERKJH4NxGmXY4GpDUhMcqpprH8rm4gzAaA2PTWNNHLcOohTTTHe6IIhrFisMP4c0cF0ahGwm2QaJlHAvqCeLvBIW0ajHMAoMHAPWc3f9PRlnYD5YfprJGGMj5WNw5h2GprKNM2ySgjMQS4TgmTwiHfhzX1cXR7MxLFOuKAjfWDLl7misK11kyb3NQq80c0awarpPkCFt5tPQmy0bd1RZO9Aa0bBk3Nf8R9K5cpdyHalOVjMwjNQiimCOCV57UQF4e3BXtibew5lvy0cPk80bkB90czaZt5MvTDF0bf6EmqDN0caWfFz3Wza8JnnqE9XbVWQKa513SkDbX2y0bkJp0cfXhxQr0bGTQIY4tHEa33buOjiStr1phJ0acpFL3wxOCUcYl43EMuoM95ccUn1fb0bwCzQwe0bZ0aC90a0cWD2dL7vg0bf9HoRPDThbwJhEYN0ageUvUaukUuONcTsOgwc6HCAQE5oBSXU7D46NBQbk0aJgr2IypUK4xfl9pfxeHXH0ag5BEMCPWa0bzi6yYEdRmmp9zCytm2gRDPh9uGre3LC6rR89Uv9dc2dZsRDN8lHIJGVimAXjFoSFz84IhVo1iCOONX2RlaE0a5aTEejx0aCGjVKJyhR8I0beqDqaqRH5QKanOjVlPTypFrfWHH3fVvs8SaoP2BDz8E3y0cufeihqJbPdpUFX4f1ISz22wxuDQRyfpWfvqY0bNym0cfFP0cP19l98Er96g84PBClWUlhn0a6k60bID0aEkTtV2bQqgj6nItsz0cnQbfZCYuHuq6MOZ1kczn9gsOxH3YITPXzeeG7J10bnVwhJE0bVxCIdO0bMIWvysmeK0aw9OaUVRyQfcHY8XqltD0aqr68A0b8KhDGdBwMMPqNdDzM8Z6vLQt9oXVZfXzO7vg4AXhdpMmvSsrIc0a0cle0cXJSZRL3wqNqmu4PXNuYaWHKdr7Pe2B5G0cCl7S0altoa0cHOjLyYKtL5K52fy0bNiurVhAdgyo4OvmYmk9Xtvb1VZM1h4FVNn2szVvtjywOZuZVi0c3AL0ay4fMbtzGWFLV81ecRmXT9wm49aPzGrh90aa0aONmF0anpcpv85Sh4W679YM0ca5h3ySmszVfZYlc0a1OaP0aJLjK5QQ2AQZ0cGAbocfibY7CeYBhw54LmYoqReJq66DyoDuJ1RKEqFRjtBs8R0cmD0aG0cwLVfHDQ8CXI4x1YBY8gx0c8v4B0b4P8c0b4NnJuH0c0bK0c6A5BCtbVC8vz0a7W9HIskxUFBzFwFokcRQQviBinuRxKjCHqIIA2BBX1Tw1qDtIOwAoB6iQxgGNio2QoAYnogjYxH8xDg6qCTcSBF9QDAk6R8kHPnrt780beeCHzQ4OeucM4eA4C2Yobg2wjTOwFPuTkN6NsKgQjmqC0cwFItZBUT8Yx4IY0bjj2akCcvswc5GPu7ad3J9F0b1RjA0bxH3QurPh9f0aYAi0bVU5oSEn1tNJ0cpXSBRfdkbmzJqy7awab1zObK1NP9oqq0b8wEjxpnL59f8AsZBVOAGKAAAEndOfSavefile
```

**Additional context**
Game is running on Oppo A53. This may be frame perfect i.e. editing when the game does a reality is causing the crash.

**Comment by lrobt97** [comment]
Trying to stop the script also causes a crash

**Comment by kajfik** [comment]
Fixed in 2.111.0i11
